BasicSteps/SweepParameterStep.cs

```diff
@@ -224,11 +230,41 @@ public override void PrePlanRun()
             base.PrePlanRun();
             iteration = 0;
 
+            if (csvSweepValues != null && csvSweepValues != "")
+            {
+                // Populates sweep tables with csv from import path
+                var timer = System.Diagnostics.Stopwatch.StartNew();
+                Type plugin = PluginManager.GetPlugins<ITableImport>().First();
```

Comment:
OpenTAP has no implementations of `ITableImport`. I assume this works for you because you have the `CSV` plugin installed, but we can't make that assumption.